aip/general/0122.md

```diff
@@ -102,17 +110,26 @@ ID for the publisher, and `les-miserables` is the resource ID for the book.
   server-generated if unset), or never set by users (not accepted at resource
   creation). They **should** be immutable once created.
   - If resource IDs are user-settable, the API **must** document allowed
-    formats.
+    formats. User-settable resource IDs **should** conform to [RFC-1034][],
```

Comment:
We've moved from "user-settable" to "user-specified" later on... should we do so here too?